auth/jwtmock_test.go

```diff
@@ -0,0 +1,99 @@
+package auth
```

Comment:
I've refactored this a bit. Added all the utility functions to this `jwtmock_test.go` and removed the unit test cases for those utility functions, since the functionality is covered while making use of these functions in different unit tests related to JWT.